cmd/mesos_state_test.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getMesosState(t *testing.T) {
+	tr := roundTripFunc(func(r *http.Request) (*http.Response, error) {
+		assert.Equal(t, r.URL.String(), "http://leader.mesos:5050/state")
+		return &http.Response{
+			StatusCode: http.StatusOK,
+			Body:       ioutil.NopCloser(strings.NewReader(`{}`)),
```

Comment:
I would return sth. more unique such as `{"this": "and that"}` to be sure the `getMesosState` function doesn't swallow the body and just always returns an empty JSON string.